platforms/hyperledger-fabric/scripts/generate-crypto.sh

```diff
@@ -110,4 +110,43 @@ while [ ${COUNTER} -lt ${NO_OF_PEERS} ]; do
 	let COUNTER=COUNTER+1
 done
 
+while [ ${COUNTER} -lt ${NO_OF_PEERS} ] && [ "$1" != "peer" ]; do
```

Comment:
is this change done for the generate-crypto tpl file?

Reply:
No. This change is just for the script as the same script is used both by the orderer and the peer and shall not include checks like $1 = 'orderer' as 'orderer' here is the name of the orderer and not the type.